src/components/backend-ai-email-verification-view.ts

```diff
@@ -0,0 +1,143 @@
+/**
+ @license
+ Copyright (c) 2015-2020 Lablup Inc. All rights reserved.
+ */
+import {get as _text, translate as _t} from "lit-translate";
+import {css, customElement, html, property} from "lit-element";
+
+import '@material/mwc-textfield/mwc-textfield';
+import 'weightless/card';
+import 'weightless/dialog';
+
+import {BackendAIPage} from './backend-ai-page';
+import {BackendAiStyles} from './backend-ai-general-styles';
+import {
+  IronFlex,
+  IronFlexAlignment,
+  IronFlexFactors,
+  IronPositioning
+} from '../plastics/layout/iron-flex-layout-classes';
+
+
+@customElement("backend-ai-email-verification-view")
+export default class BackendAIEmailVerificationView extends BackendAIPage {
+  @property({type: Object}) consoleShell = Object();
+  @property({type: Object}) clientConfig = Object();
+  @property({type: Object}) client = Object();
+  @property({type: Object}) notification = Object();
+  @property({type: Object}) successDialog = Object();
+  @property({type: Object}) failDialog = Object();
+
+  static get styles() {
+    return [
+      BackendAiStyles,
+      IronFlex,
+      IronFlexAlignment,
+      IronFlexFactors,
+      IronPositioning,
+      // language=CSS
+      css`
+        mwc-textfield {
+          width: 100%;
+        }
+      `
+    ];
+  }
+
+  _initClient(apiEndpoint: string) {
+    this.consoleShell = document.querySelector('#console-shell');
+    this.consoleShell.appBody.style.visibility = 'visible';
```

Comment:
Why do we need this part? This part produces FOUC issue occasionally.

Reply:
Without this, verification dialogs are not visible to users. Is there other workaround to make dialogs visible?

Reply:
Ok. It is only called while email verification is processed. Let's see after merging.